### DISCUSSION TITLE
Unit 11 - Zero-shot Computer Vision

### DISCUSSION THREAD

**Discussion by mmhamdy** [huggingface/computer-vision-course#96] [11. Zero Shot Computer Vision]
This is the outline of Unit 11 which will discuss zero-shot learning in computer vision. Each section represents a chapter in this unit (similar to other HF courses).

The issue for the draft outline has some more details, check it out here #43

- [x] Introduction (assigned to @mmhamdy)
    - On Generalization
    - Zero-shot Learning (ZSL), History and Definitions
    - Comparison With Other Techniques
    - Relationship with Transfer Learning
- [ ] Zero-shot Learning (assigned to @albertkao227)
    - ZSL vs. Generalized ZSL
    - Semantic Information
    - Embedding Spaces
    - ZSL Algorithms
    - Benchmarks
- [ ] Zero-shot Image Recognition (assigned to @mmhamdy)
    - Methods (candidates are: TransZero, ViT-ZSL)
- [ ] Zero-shot Object Detection (assigned to @Maddy011 )
    - Methods (candidates are: OWL-ViT)
- [ ] Zero-shot Image Segmentation
    - Methods (candidates are: CLIPSeg)
- [ ] Challenges and Frontiers
    - Advantages of Zero-shot Learning
    - Applications of Zero-shot Learning
    - Challenges and Limitations of Zero-shot Learning
    - Frontiers
- [ ] Summary

**Comment by maddy011** [comment]
Heyy @mmhamdy 

Can I work on Zero-shot Object Detection and Segmentation if it's not taken already? 
If yes, then let me know how you're planning to structure the Zero-shot Image Recognition part so that all three parts can be uniform.

Thank you :)

**Reply by mmhamdy** [reply]
Cool! I just assigned you to it.

**Comment by seshupavan** [comment]
Hello @mmhamdy, the commits for unit-11 have been quite for a while compared to the syllabus. May i know the reason? is it because of a inactive team?

**Reply by mmhamdy** [reply]
Hi @seshu-pavan. The only active members in this group are me and @albertkao227. The 2nd section is almost done, and the 3rd and last 2 are WIP. other than that, are unassigned which I'm planning to work on once I finish WIP.

Thanks for asking by the way.

**Reply by seshupavan** [reply]
There is nothing you can do if others are inactive but thanks for holding this far. Can you just copy this response and post there so everything stays align: https://github.com/johko/computer-vision-course/discussions/185

**Reply by mmhamdy** [reply]
Sure!
